app/components/articleSearch/articleSearch.scss

```diff
@@ -1,17 +1,17 @@
 @import "../../variables";
 
 .articleSearchContainer {
-  margin: $navbar-height auto 0 auto;
-  max-width: $container-width;
+  margin: $navbar_height auto 0 auto;
+  max-width: $container_width;
   position: relative;
 
   .innerContainer {
-    max-width: $search-container-width;
+    max-width: $search_container_width;
```

Comment:
Why do u choose to use underscore rather than dash?

Reply:
I want to get unity at scss variables.